docs/2020/registration.md

```diff
@@ -1,9 +1,5 @@
 # OSG All-Hands Meeting 2020 &ndash; Registration
 
-## Costs
+Registration is free, but required.
```

Comment:
Participation in the event will be free, but we plan to ask participants to register in advance.